riscv/csrs.cc

```diff
@@ -1255,6 +1258,7 @@ reg_t dcsr_csr_t::read() const noexcept {
   v = set_field(v, DCSR_CAUSE, cause);
   v = set_field(v, DCSR_STEP, step);
   v = set_field(v, DCSR_PRV, prv);
+  v = set_field(v, CSR_DCSR_V, v);
```

Comment:
v here is an internal variable, and does not refer to the class member v as intended.
```suggestion
  v = set_field(v, CSR_DCSR_V, this->v);
```

(Great example of how even people don't even notice obvious bugs when reviewing code.)

Reply:
Yikes.